config/crds/OWNERS

```diff
@@ -0,0 +1,10 @@
+# Disable inheritance as this is an api owners file
+options:
+  no_parent_owners: true
+filters:
+  ".*":
+    approvers:
+      - api-approvers
+  ".*\\.yaml":
```

Comment:
```suggestion
  ".*\\.yaml*":
```

Reply:
to catch the yaml-patch too.

Reply:
technically not necessary because the yaml-patch influences the yaml.

Reply:
I think this will still catch `.yaml-patch`, I gave it a test in https://go.dev/play/p/SWvd6UvJj2b  because I am terrible at regex

Reply:
oh, you are right because there is no `$` at the end